### DISCUSSION TITLE
Any way to control the cameras using this integration?

### DISCUSSION THREAD

**Discussion by joshmd** [mrlt8/docker-wyze-bridge#302] [Ideas]
I'm currently using this repo: https://github.com/JoshuaMulliken/ha-wyzeapi to turn my cameras on and off based on specific automations in Home Assistant. Is there a way to pull that functionality into this somehow? That way I wouldn't need both :)

**Comment by mrlt8** [comment]
I believe ha-wyzeapi relay commands over the wyze servers to communicate with the cameras, whereas this project uses TUTK to communicate directly with the cameras locally. We may be able to do something if we can find the iotc commands for power on/off.

**Reply by joshmd** [reply]
Thanks!
